src/components/Table/TableHead/TableHeaderSortable/TableHeaderSortable.test.jsx

```diff
@@ -37,7 +37,7 @@ describe('TableHead', () => {
 
         const sortableColumnHeader = screen.getByText('ID');
 
-        fireEvent.keyDown(sortableColumnHeader, { key: 'Enter', code: 'Enter' });
+        fireEvent.keyDown(sortableColumnHeader, { keyCode: 'Enter', code: 'Enter' });
```

Comment:
Does this work? It doesn't match the docs?

https://testing-library.com/docs/dom-testing-library/api-events